scripts/Muon/GUI/MuonAnalysis/plot_widget/plot_time_fit_pane_model.py

```diff
@@ -0,0 +1,61 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+from Muon.GUI.Common.plot_widget.data_pane.plot_group_pair_model import PlotGroupPairModel
+from Muon.GUI.Common.ADSHandler.workspace_naming import *
+
+
+class PlotTimeFitPaneModel(PlotGroupPairModel):
+
+    def __init__(self, context):
+        super().__init__(context,"Fit Data")
+        self.context.plot_panes_context[self.name].set_defaults([0.,15.], [-0.3, 0.3])
```

Comment:
Can we get these defaults from the fitting context?

Reply:
where are they in the fitting context? I know things have moved around

Reply:
Oops, I think I meant the MuonContext. We have the default end x stored here, so maybe we should put in variables for the default start x and default y range
https://github.com/mantidproject/mantid/blob/3a31ba7409a9cf020de27f56930ab113e2473654/scripts/Muon/GUI/Common/contexts/data_analysis_context.py#L46
https://github.com/mantidproject/mantid/blob/3a31ba7409a9cf020de27f56930ab113e2473654/scripts/Muon/GUI/Common/contexts/frequency_domain_analysis_context.py#L34

Reply:
I think we should just import the end_x. We can assume for plotting it starts at t=0 and the y range is purely visual so should be in the plotting pane

Reply:
we cant do this for the data_pane, as the end_x value for the FDA will return 1000 and not 15 as expected. So we will need to hard code the 15.